app/view_models/prior_authority/v1/application_details.rb

```diff
@@ -97,7 +97,11 @@ def section_amended?(provider_section_name)
       def further_information_cards
         return [] unless submission.data['further_information']
 
-        submission.data['further_information'].select { _1['information_supplied'].present? }.map do |further_information|
+        submission.data['further_information']
+                  .select { _1['information_supplied'].present? }
+                  .sort_by { DateTime.parse(_1['requested_at']) }
```

Comment:
don't think you need to sort JSON dates as string sort is good enough (maybe check)